apps/meteor/app/importer/server/classes/converters/generateTempPassword.ts

```diff
@@ -0,0 +1,5 @@
+import type { IImportUser } from '@rocket.chat/core-typings';
```

Comment:
We had to extract this code into a function (`generateTempPassword`) so that we could mock it in our unit tests and make sure the password is being correctly handed when it is not explicitly provided

Reply:
CI shenanigans, we had to do it to avoid losing a(nother) whole day figuring out wtf was going on. For some reason our PR causes sinon to no longer stub class functions properly, but only on the CI machine. I'll still look into it later; I know I'm not sleeping well until I figure it out.

Reply:
Interesting 🤔